autoload/lsp/omni.vim

```diff
@@ -199,7 +213,8 @@ function! lsp#omni#default_get_vim_completion_item(item, ...) abort
     if l:do_remove_typed_part
         let l:word = s:remove_typed_part(l:word)
     endif
-    let l:kind = lsp#omni#get_kind_text(a:item)
+
+    let l:kind = lsp#omni#get_completion_item_kind_text(a:server_name,a:item)
```

Comment:
This might help ;)
```suggestion
    let l:kind = lsp#omni#get_completion_item_kind_text(a:server, a:item)
```

Reply:
No, it doesn't -- sorry, the commit was a hip shot from my working (hard coded) state; when I tested it, I used the correct name, of course :)